src/modules/Header.jsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import ReplyIcon from '@material-ui/icons/Reply';
+import FiberNewOutlinedIcon from '@material-ui/icons/FiberNewOutlined';
+import FullscreenIcon from '@material-ui/icons/Fullscreen';
+import MenuIcon from '@material-ui/icons/Menu';
+import { getInitialBoardArray, setValueInLocalStorage } from './utils/helpers';
+
+const Header = (props) => {
+  const { setIsPopUpActive, bestScore, boardSize, history, setHistory, stepBackButton } = props;
```

Comment:
you can do it in 1 line: `const Header = ({all, the, props, you, need, ...rest if you don't know what else should be passed}) => {...}`

Reply:
Yeah I know that, don't know why I didn't use it though))